hazelcast/src/main/java/com/hazelcast/internal/nio/ConnectionType.java

```diff
@@ -77,6 +77,11 @@
      */
     public static final String MEMCACHE_CLIENT = "MEMCACHE";
 
+    /**
+     * Management Center Java client Connection Type
+     */
+    public static final String MC_JAVA_CLIENT = "MCCLIENT";
```

Comment:
would it ever be possible that MCCLIENT may be java or non-java for example Node.js. Then, we may choose a better name to reflect it.

Reply:
I don't think it's very likely to happen, but I'm going to change the constant to `MCJVM` just in case. :)